hint.base.css

```diff
@@ -1,4 +1,4 @@
-/*! Hint.css (base version) - v1.3.5 - 2015-06-16
```

Comment:
@mattconvente Please remove this any built CSS file for the PR. I'll build this when doing final merge.